src/components/Error.tsx

```diff
@@ -39,7 +39,7 @@ export const Error = ({ heading, children, ...props }: ErrorPropTypes) => {
   return <ModalBody {...props} data-testid='error'>
     <ModalBodyHeading>{heading ?? `Uh-oh, there's been a glitch`}</ModalBodyHeading>
     {children ?? <>
-      We're not quite sure what went wrong. Restart your browser. If this doesn't solve
+      We&apos;re not quite sure what went wrong. Restart your browser. If this doesn&apos;t solve
```

Comment:
is there a config we can change so we don't have to do this?

Reply:
This is for the ["no-unescaped-entities" rule](https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/no-unescaped-entities.md). It is turn-off-able. Its purpose is to catch where markup intended to be JSX gets included in text.
As an alternative to the HTML entity, I think we could use `{'}` for the apostrophe.

Reply:
hmm how do we handle the rule on the other projects?

Reply:
It doesn't look like we have the react plugins for eslint in rex or assignments.
I turned off the no-unescaped-entities rule.